django_app/redbox_app/redbox_core/models.py

```diff
@@ -209,6 +209,9 @@ def expires_at(self) -> datetime:
     def expires(self) -> timedelta:
         return self.expires_at - datetime.now(tz=UTC)
 
+    def __lt__(self, other):
+        return self.id < other.id
```

Comment:
Would `created_at` be a better metric to sort by? Or does the order not matter?

Reply:
Right now, it doesn't matter - we only need a total ordering so that we can group citations by file. If one day we need some specific order, we can revisit.

Reply:
I've put in another comment about the grouping by file - can we preserve the order of the 'most relevant file'? Or is that for a later PR?

Reply:
Right now we don't store the citation order at all AFAICT. We might need to revisit that.